pontoon/administration/templates/admin_project.html

```diff
@@ -259,6 +259,7 @@ <h3>Project info <span class="small stress">(optional)</span></h3>
     <option value="Production build">
     <option value="Screenshots">
     <option value="Language pack">
```

Comment:
Can we remove "Language Pack" at this point? As far as I can tell, that's not supported anywhere at this point.